### DISCUSSION TITLE
TypeError: put() got an unexpected keyword argument 'expire_in'

### DISCUSSION THREAD

**Discussion by xu2xulim** [deta/cloud-docs#229] [Help] [answered]
I have been using expire_in as part of a put but today I have not been able to get it to work.

log = Deta().Base('production_run_log')
Without the expire_in the update is successful

log.put(item, expire_in = 86400)
TypeError: put() got an unexpected keyword argument 'expire_in'

**Comment by abdelhai** [comment, marked as answer]
hi, make sure to update the deta sdk to the latest version.

**Reply by xu2xulim** [reply]
Thanks. Got it to work.
